conda/third_party/jpeg_turbo/recipe/build.sh

```diff
@@ -1,44 +0,0 @@
-#!/bin/bash
```

Comment:
I was not sure whether conda-forge provides Power packages. In any case, I believe this falls outside of our repository. Having a valid libjpeg-turbo package available (like the one available in conda-forge) doesn't have to be here. I can bring it back, but in my opinion, it will only make the situation worse when we will have to maintain custom ffmpeg, opencv, and more